examples/GetAccountList/Program.cs

```diff
@@ -0,0 +1,53 @@
+﻿using CommandLine;
+using Concordium.Sdk.Client;
+using Concordium.Sdk.Types;
+
+#pragma warning disable CS8618

```

Comment:
I have removed it. It is there because one should initialize the properties but CommandLine isn’t using the defaults either way.

Since these are example I think it is fine as is.

Reply:
I have added it back again - I believe it would be nice to have the compiler warnings removed. 

We don’t want to provide defaults but the command line parser needs to have setters to populate the properties.